table/tables/partition.go

```diff
@@ -1590,6 +1599,14 @@ func partitionedTableUpdateRecord(gctx context.Context, ctx sessionctx.Context,
 			return errors.WithStack(table.ErrRowDoesNotMatchGivenPartitionSet)
 		}
 	}
+	// TODO(L-Maple): Handle global index.
```

Comment:
@mjonss onDropPartition is affected by onReorganizePartition. 
1.When a table has a global index, `DroppingDefinitions` field is also set. Maybe we ignore it in the code logic.
2.For example, pt has p0,p1,p2 and p0 is in dropPartition reorg. If we write to p0, `to` is the pid of p1. That's actually wrong.